### ISSUE TITLE
TimeStep classes not exported?

### ISSUE THREAD

**Issue by torreyleonard** [BrainJS/brain.js#199]
> "If you don't mind add a fun gif or meme, but no pressure"

![A GIF or MEME to give some spice of the internet](https://media.giphy.com/media/AIoAyHX5j59G8/giphy.gif)

I know this feature was released just a few days ago so I thought I'd open this issue to help with its progress, although this could just be me overlooking something.

Instantiating a new LSTMTimeStep returns the error below, and `brain.recurrent.LSTMTimeStep` is `undefined.` I fixed this by exporting it in the compiled index.js, then instantiating with``` .default():```
```js
// Before

const net = new brain.recurrent.LSTMTimeStep.(options);

// const net = new brain.recurrent.LSTMTimeStep({
//             ^
// TypeError: brain.recurrent.LSTMTimeStep is not a constructor


// After

const net = new brain.recurrent.LSTMTimeStep.default(options);

// dist/index.js
var LSTMTimeStep = require('./dist/recurrent/lstm-time-step');
var brain = {
  crossValidate: crossValidate,
  likely: likely,
  lookup: lookup,
  NeuralNetwork: NeuralNetwork,
  NeuralNetworkGPU: NeuralNetworkGPU,
  TrainStream: TrainStream,
  recurrent: {
    RNN: RNN,
    LSTM: LSTM,
    GRU: GRU,
    LSTMTimeStep: LSTMTimeStep // <---
  },
  utilities: utilities
};
```

I'm sure that this problem would apply to the other "TimeStep" classes as well, but I've only tested it on ```RNNTimeStep``` which does indeed produce the same error.

**Comment by robertleeplummerjr** [comment]
How are you importing brain?

**Comment by torreyleonard** [comment]
Using ```const brain = require('brain.js');```

**Comment by thingbuilder** [comment]
I get the same error, my code is:
```
const brain = require('brain.js');
const net = new brain.recurrent.LSTMTimeStep();
net.train([1,2,3]);
var output = net.run([1, 2]);  // 3
console.log(output);
```
error:
```
const net = new brain.recurrent.LSTMTimeStep();
            ^

TypeError: brain.recurrent.LSTMTimeStep is not a constructor
```

if I use `const brain = require('brain.js').recurrent;`
I get this error:
```
const net = new brain.recurrent.LSTMTimeStep();
                               ^

TypeError: Cannot read property 'LSTMTimeStep' of undefined
```

**Comment by robertleeplummerjr** [comment]
Released v1.2.1 to address this.  Ty for finding.

![](https://i.pinimg.com/originals/c3/62/a5/c362a51974a87d310110c9141dfa5d77.gif)
